lib/car.rb

```diff
@@ -0,0 +1,17 @@
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :total_cost, :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    make_and_model = make_model.split
+    @make = make_and_model.first
+    @model = make_and_model.last
```

Comment:
```suggestion
    @make = make_model.split.first
    @model = make_model.split.last
```

If you're going to perform this calculation right in the `initialize` method (which is fine, it's how I did it too), I would probably skip the var assignment on line 5 and just split on lines 6 and 7. Is it marginally less efficient? Probably. But defining a local variable in `initialize` seems like the greater offense against ruby convention.